typescript/infra/src/config/agent/scraper.ts

```diff
@@ -9,8 +10,7 @@ import { HelmStatefulSetValues } from '../infrastructure.js';
 import { AgentConfigHelper, RootAgentConfig } from './agent.js';
 
 export interface BaseScraperConfig {
-  // no configs at this time
-  __placeholder?: undefined;
+  scraperOnlyChains?: ChainMap<boolean>;
```

Comment:
why ChainMap? do we ever honor the boolean being true or false?

Reply:
resolved in 132fa02a5c0545f1020358c69da32691ce96c5d3